CHANGELOG.md

```diff
@@ -62,4 +63,4 @@ to set the CSS class of dropdown item.
 - Text from Admin's toString method is escaped for html output before adding in flash message to prevent possible XSS vulnerability.
 
 ### Removed
-- Remove `btn-outline` from doctrine-orm-admin form actions buttons
+- Remove `btn-outline` from doctrine-orm-admin form actions buttons
```

Comment:
Please configure your editor to add a newline at end of file  (will make the red pictogram disappear)